src/vivarium_cluster_tools/vipin/perf_report.py

```diff
@@ -105,26 +105,26 @@ def set_index_scenario_cols(perf_df: pd.DataFrame) -> Tuple[pd.DataFrame, list]:
     return perf_df, scenario_cols
 
 
-def add_jobapi_data(perf_df: pd.DataFrame):
-    """Given a dataframe from PerformanceSummary.to_df, add QPID Job API data for the job.
-    Job API reference: https://stash.ihme.washington.edu/projects/QPID/repos/job-db/browse/docs/index.md
+def add_squid_api_data(perf_df: pd.DataFrame):
+    """Given a dataframe from PerformanceSummary.to_df, add Squid API data for the job.
+    Squid API reference: https://hub.ihme.washington.edu/display/SCKB/How+to+use+Squid+API
     """
     try:
         job_numbers = perf_df["job_number"].unique()
         assert len(job_numbers) == 1
-        jobapi_data = requests.get(
-            "http://jobapi.ihme.washington.edu/fair/queryjobids",
-            params=[("job_number", job_numbers[0]), ("limit", 50000)],
+        print(job_numbers)
```

Comment:
Is print the way we want to log this? Also, shouldn't there be some context provided with the list of job numbers?

Reply:
ugh, no that's debug code left in.  Let me get rid of it.

Reply:
Good eye!